packages/runtime/src/EnsembleApp.tsx

```diff
@@ -62,13 +68,23 @@ export const EnsembleApp: React.FC<EnsembleAppProps> = ({
           ],
         },
       ]),
-    [app],
+    [app]
   );
+
   return (
     <ApplicationContextProvider app={app}>
       <ThemeProvider>
-        <RouterProvider router={router} />
-        <ToastContainer />
+        {oAuthClientId ? (
+          <GoogleOAuthProvider clientId={oAuthClientId}>
```

Comment:
Have you tried just moving this into the component? Looking through the source for this library, I don't see any reason why this has to be at the top of the app